model_signing/serialization/serialize_by_file.py

```diff
@@ -95,15 +107,18 @@ def __init__(
               hash individual files.
             max_workers: Maximum number of workers to use in parallel. Default
               is to defer to the `concurrent.futures` library.
+            allow_symlinks: Controls whether symlinks are serialized or raise
+              a ValueError. Default is to disallow symlinks.
```

Comment:
Let's include a `Raises` section.

Also, nit: "Controls whether symbolic link are included. If a symlink is present but the flag is False (default) the serialization would raise an error."

Reply:
Addressed the nit.
With regards to the `Raises` section, the __init__() doesn't raise the error, is it still relevant to include?

Reply:
Oh, I missed this was on the `__init__`. It shouldn't be there, only when this gets raised.

Reply:
I'll add this to the various serializer methods

Reply:
Sorry, I made it worse when I tried to resolve the conflict from the web interface. Apparently that doesn't insert a newline at the end of the file 🤦

Reply:
Force pushed away :)

Please take another look at the docstrings in 053de6f43037af4224f8453f295a030ee3352b8d

Reply:
Nice! You covered the ones that are a method call away (via `super()`, etc.). Well done!